src/uct/cuda/base/cuda_md.c

```diff
@@ -10,7 +10,8 @@
 #include <cuda.h>
 
 
-int uct_cuda_is_mem_type_owned(uct_md_h md, void *addr, size_t length)
+ucs_status_t uct_cuda_base_detect_memory_type(uct_md_h md, void *addr, size_t length,
```

Comment:
The only UCT object that you pass here is uct_md_h and you are not using it ?

Reply:
Seems like the same true for ROCM

Reply:
this is uct_ms_ops func. This is consistent with all other ops prototype (https://github.com/openucx/ucx/blob/master/src/uct/base/uct_md.h#L62) .

Reply:
So what do we have this arg all together ?

Reply:
@yosefe  can you comment on here?

Reply:
for current implementation it's not used, but IMHO it makes sens that some other MD in the future may need it